mockito-kotlin/src/main/kotlin/com/nhaarman/mockitokotlin2/OngoingStubbing.kt

```diff
@@ -124,3 +126,14 @@ infix fun <T> OngoingStubbing<T>.doAnswer(answer: Answer<*>): OngoingStubbing<T>
 infix fun <T> OngoingStubbing<T>.doAnswer(answer: (InvocationOnMock) -> T?): OngoingStubbing<T> {
     return thenAnswer(answer)
 }
+
+infix fun <T> OngoingStubbing<T>.willAnswer(answer: suspend (InvocationOnMock) -> T?): OngoingStubbing<T> {
```

Comment:
Thoughts on naming this `doAnswer`? The compiler will consider it as a valid overload I believe.

Reply:
I believe the `willAnswer` variants are extensions for `BDDMyOngoingStubbing`. May I request that we also add `suspend` support for `BDDMyOngoingStubbing`? 🙂

The tests would look something like:
```Kotlin
...
given(fixture.suspendingWithArg(any())).willAnswer {
    withContext(Dispatchers.Default) { it.getArgument<Int>(0) }
}
...
```
Thank you!

Reply:
@TimvdLippe what do you think about this request? I could create `willSuspendableAnswer` for BDD as well.

Reply:
We currently do not have any extensions for `BDDMyOngoingStubbing` in the existing `OngoingStubbing.kt`. I think we should treat that as a separate improvement, but I would happy to approve a PR on that. For now, let's keep that separate and merge this PR as-is.